src/librustc_expand/expand.rs

```diff
@@ -1389,7 +1387,9 @@ impl<'a, 'b> MutVisitor for InvocationCollector<'a, 'b> {
                     push_directory(ident, &item.attrs, dir)
                 } else {
                     // We have an outline `mod foo;` so we need to parse the file.
-                    let (new_mod, dir) = parse_external_mod(sess, ident, dir, &mut attrs, pushed);
+                    let span = sess.source_map().def_span(span);
```

Comment:
This is certainly an out-of-line module, so `def_span` is unnecessary.
(It works like "this span but only until the first `{`".)

Reply:
Oh right; forgot about that. 😂

Reply:
Done.